app/components/submission/Mutations.js

```diff
@@ -0,0 +1,93 @@
+import gql from 'graphql-tag'
+
+const manuscriptFragment = gql`
+  fragment WholeManuscript on Manuscript {
+    id
+    title
+    manuscriptType
+    subjectAreas
+    suggestedSeniorEditors
+    opposedSeniorEditors {
+      name
+      reason
+    }
+    suggestedReviewingEditors
+    opposedReviewingEditors {
+      name
+      reason
+    }
+    suggestedReviewers {
+      name
+      email
+    }
+    opposedReviewers {
+      name
+      email
+      reason
+    }
+    noConflictOfInterest
+    files {
+      name
+      type
+    }
+    submissionMeta {
+      coverLetter
+      author {
+        firstName
+        lastName
+        email
+        institution
+      }
+      stage
+      discussion
+      previousArticle
+      cosubmission {
+        title
+      }
+    }
+    manuscriptPersons {
+      role
+      metadata {
+        ... on AuthorMetadata {
+          confirmed
+        }
+      }
+    }
+  }
+`
+
+export const GET_CURRENT_SUBMISSION = gql`
+  query CurrentSubmission {
```

Comment:
This file contains both queries and mutations. I don't think it's worth splitting them into different files. Perhaps `operations.js` would be a better name for the file?

Reply:
sgtm

Reply:
In other places, we name the file `queries.js`, which I think is more apt.